theme/static/js/custom.js

```diff
@@ -222,23 +222,37 @@ $(document).ready(function () {
         });
     }
 
+    function popup_profile_alert(uid, fields) {
+        if ($("#publish").length) {
+                $("#publish").toggleClass("disabled", true);
+                $("#publish").removeAttr("data-toggle");   // Disable the agreement modal
+                $("#publish > [data-toggle='tooltip']").attr("data-original-title",
+                    "Your profile information must be complete before you can formally publish resources.");
+            }
+            missing_field_str = fields.join(', ')
+
+            var message = 'Your profile is nearly complete. Please fill in the '
+                + '<strong>' + missing_field_str + '</strong> fields'
+                + ' on the <a href="/user/' + uid + '/">User Profile</a> page';
+
+            customAlert("Profile", message, "info", 10000);
+    }
+
     $.ajax({
         url: "/hsapi/userInfo/",
         success: function(user) {
-            if(!user.organization) {
+            var missing_profile_fields = [];
+            if(!user.organization)
```

Comment:
not necessary to fix here, but going forward after Resource Landing Page is cleaned up, we'll be more disciplined around var, let, const and variable scopes in general. Linting will also be more strict.

Reply:
Got it and sounds good.